tests/server/tso/tso_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 TiKV Project Authors.
+// Copyright 2024 TiKV Project Authors.
```

Comment:
Why change it?

Reply:
The original content of `tso_test.go` has been modified, and the test cases from `global_tso_test.go` have been added, so I simply consider it a new file.